plugins/3MFReader/WorkspaceDialog.qml

```diff
@@ -103,12 +103,17 @@ UM.Dialog
                     comboboxTitle: catalog.i18nc("@action:label", "Open With")
                     comboboxTooltipText: catalog.i18nc("@info:tooltip", "Printer settings will be updated to match the settings saved with the project.")
                     comboboxVisible: workspaceDialog.visible && manager.updatableMachinesModel.count > 1
```

Comment:
Now that the `combobox` property isn't being used anymore any logic that was previously handled through the `WorkspaceSection` component doesn't work anymore. The solution implemented here doesn't work for me
```
2024-03-18 11:30:56,354 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [448]: file:///Users/c.lamboo/ultimaker/Cura/plugins/3MFReader/WorkspaceDialog.qml:122: ReferenceError: comboboxVisible is not defined
```
Do we know why the behavior is different when the component is passed via the `combobox` property? I would solve a lot of duplicate code, things like positioning and visibility are already handled in that component which is now implemented twice.

Reply:
I believe the two mouse area of combobox and machineselector creates a conflict of interest thats why we get this issue. We still use combobox below in the code for materialResolveComboBox